mainnet/3a6f7a95addee6634463e8a264d8aa07baf656d45d7fc63f5bf8d2f078182215.json

```
@@ -0,0 +1,10 @@
+{
+  "id": 62,
+  "name": "Forest Staking",
+  "secp": "3a6f7a95addee6634463e8a264d8aa07baf656d45d7fc63f5bf8d2f078182215",
```

The secp key seems incorrect.

```suggestion
  "secp": "021f68fac315a1b9e7258af85385b780e7a594819d7532015182a74b80442e71e6",
```